package.json

```diff
@@ -6,7 +6,8 @@
   "main": "index.js",
   "module": "index.js",
   "scripts": {
-    "test": "eslint *.js"
+    "test": "eslint *.mjs",
+    "bench": "node --experimental-modules bench.mjs"
```

Comment:
For Node v8+ support, let's do `node -r esm bench.js` instead. The dep is already there.